[2025-09-21T11:38:01.769+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:38:01.769+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:38:01.808+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:38:01.809+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:38:01.862+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:37:00+00:00 [queued]>
[2025-09-21T11:38:01.873+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:37:00+00:00 [queued]>
[2025-09-21T11:38:01.874+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:38:01.891+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-09-21 11:37:00+00:00
[2025-09-21T11:38:01.903+0000] {standard_task_runner.py:57} INFO - Started process 1209 to run task
[2025-09-21T11:38:01.910+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-09-21T11:37:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmp32xg4ujy']
[2025-09-21T11:38:01.915+0000] {standard_task_runner.py:85} INFO - Job 104: Subtask fetch_training_data
[2025-09-21T11:38:01.984+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:38:01.985+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:37:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:38:01.986+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:37:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:38:02.085+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:37:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:37:00+00:00'
[2025-09-21T11:38:02.096+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-09-21T11:38:03.258+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100554  155994     77  ...    product_A 2025-09-21 11:18:16.078203
100555  155995     76  ...    product_A 2025-09-21 11:18:26.123748
100556  155996     75  ...    product_A 2025-09-21 11:18:36.094021
100557  155997     74  ...    product_A 2025-09-21 11:18:46.999195
100558  155998     73  ...    product_A 2025-09-21 11:18:57.095432

[5 rows x 12 columns]
[2025-09-21T11:38:03.345+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-06-23', 'end_date': '2025-09-21'}
[2025-09-21T11:38:03.414+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250921T113700, start_date=20250921T113801, end_date=20250921T113803
[2025-09-21T11:38:03.491+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:38:03.520+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
